dictionaries/xp_levels.json

```diff
@@ -0,0 +1,32 @@
+{
+  "Level 1": "0",
+  "Level 2": "150",
+  "Level 3": "300",
+  "Level 4": "650",
+  "Level 5": "1100",
+  "Level 6": "1600",
+  "Level 7": "2200",
+  "Level 8": "2900",
```

Comment:
same question as above ignore these comments if im wrong that they are supposed to be this way